libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    border-top: solid 1px #f3f3f3;
+  }
+
+  .feds-links-strip {
+    width: 100%;
+    padding: 20px 0;
+    max-width: var(--feds-maxWidth-nav);
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .feds-links-strip > *:not(:last-child)::after {
+    content: '|';
+    margin: 0;
+  }
+
+  .feds-links-strip-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
I see that the `linksStrip-item` contains a decorated `navLink`, which already has this setting, this might be redundant.

Reply:
This is necessary for aligning the `|` pseudo-element with the `navLink` inside the `linksStrip-item`. Do you think this can be done with less css ?